pages/create_post_page.py

```diff
@@ -10,12 +12,26 @@ class CreatePostPage(BasePage):
     def __init__(self, driver):
         super().__init__(driver)
         self.const = CreatePostPageConsts
-        self.log = logging.getLogger("[CreatePostPage]")
+        self.header = Header(self.driver)
+
+    @wait_until_ok(timeout=3, period=0.5)
+    def click_and_validate_save_post_button(self):
+        """Click on Save Post button until it disappear"""
+        self.click(self.const.SAVE_POST_BUTTON_XPATH)
+        assert not self.is_element_exists(self.const.SAVE_POST_BUTTON_XPATH), "Save Post button didn't disappear"
 
     def create_post(self, title, body):
         """Create post using provided values"""
         self.fill_field(xpath=self.const.TITLE_INPUT_XPATH, value=title)
         self.fill_field(xpath=self.const.BODY_AREA_XPATH, value=body)
+        # Click on Save Post button
+        if verify:
```

Comment:
I think you don't need this logic here. But if I wrong, you should add additional argument to the method and pass it from the test